### PR TITLE
Pycon jamaica 2016

### PR DESCRIPTION
**Pull request by Daniel-at-github**
Closes issue #132
and reserializes repository

### CODE REVIEW

File: pycon-jamaica-2016/videos/a-non-coders-journey-with-python-and-postgresql-rhys-stewart.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": "Python is sometimes thought of as the language for the casual coder. Rhys Stewart, self declared \"non-coder\" and SQL junkie (which means he can do crazy things with databases) will share his experiences of using Python with Postgresql for day to day tasks. He's not promising best practices but he says they are \"pretty good\" and work for him.\n\nhttps://www.pythonjam.org.jm/conference-2016 the PyCon Jamaica 2016 technology conference for Python software developers was held (November 17 - 18, 2016).  The conference was immediately followed by the Great Python Scavenger Hunt https://www.pythonjam.org.jm/conference-2016/scavenger-race. Our conferences are open to beginners, experts and non-programmers.",
+  "duration": 2851,
+  "language": "eng",
+  "recorded": "2016-11-25",
```
Comment (redapple) [inline pycon-jamaica-2016/videos/a-non-coders-journey-with-python-and-postgresql-rhys-stewart.json]: Looks like this was on 2016-11-18 (https://www.pythonjam.org.jm/conference-2016/schedule)

File: pycon-jamaica-2016/videos/pycon-jamaica-2016-diy-serverless-platform-with-python3-and-docker-joir-dan-gumbs.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": "A walkthrough on how I built my own serverless platform to run both ephemeral and long-lasting functions in Python on top of Docker, capable of handling REST and websocket connections. Will go over architecture, show code, and discuss pain points as well as next steps.  #pyconjamaica2016",
+  "duration": 2690,
+  "language": "eng",
+  "recorded": "2016-11-29",
```
Comment (redapple) [inline pycon-jamaica-2016/videos/pycon-jamaica-2016-diy-serverless-platform-with-python3-and-docker-joir-dan-gumbs.json]: Was on 2016-11-18

File: pycon-jamaica-2016/videos/pycon-jamaica-2016-is-that-ham-in-my-spam-a-novices-inquiry-into-naive-bayes-lorena-mesa.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "duration": 1950,
+  "language": "eng",
+  "recorded": "2016-11-24",
```
Comment (redapple) [inline pycon-jamaica-2016/videos/pycon-jamaica-2016-is-that-ham-in-my-spam-a-novices-inquiry-into-naive-bayes-lorena-mesa.json]: Was on 2016-11-18

File: pycon-jamaica-2016/videos/snakes-on-an-ambulance-python-at-the-ministry-of-health-marc-murray.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": "An exploration of how Python is being used at the Ministry of Health, Jamaica to improve the overall patient experience.\n\nhttps://www.pythonjam.org.jm/conference-2016 the PyCon Jamaica 2016 technology conference for Python software developers was held (November 17 - 18, 2016).  The conference was immediately followed by the Great Python Scavenger Hunt https://www.pythonjam.org.jm/conference-2016/scavenger-race. Our conferences are open to beginners, experts and non-programmers.",
+  "duration": 2481,
+  "language": "eng",
+  "recorded": "2016-11-25",
```
Comment (redapple) [inline pycon-jamaica-2016/videos/snakes-on-an-ambulance-python-at-the-ministry-of-health-marc-murray.json]: Was on 2016-11-18

**Comment by logston** [comment]
👍

**Pull request merged**